test/entities/swapV3.integration.test.ts

```diff
@@ -40,7 +40,7 @@ import { Path } from '@/entities/swap/paths/types';
 const vaultVersion = 3;
 const chainId = ChainId.SEPOLIA;
 // blockNo shouldn't change as checks depend on token balances
-const blockNo = 6001201n;
+const blockNo = 6178561n;
```

Comment:
Set to a few blocks after the last mock pool deployment since swaps involve all 4 mock pools

Reply:
this block number was too soon :facepalm: